BlitzBoard/Features/HomeScreenTableCell/HomeScreenTableViewCellViewController.swift

```diff
@@ -0,0 +1,33 @@
+//  HomeScreenTableViewCellViewController.swift
+//  BlitzBoard
+//
+//  Created by Kiasha Rangasamy on 2024/04/17.
+//
+
+import UIKit
+
+class HomeScreenTableViewCellViewController: UITableViewCell {
```

Comment:
Please rename this file to HomeScreenTableViewCell. It is not a view controller so it does not need to have ViewController in the name.
Please then rename the class to HomeScreenTableViewCell and the comment at the top